dev/conductor/ui/lib/widgets/progression.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+import 'package:conductor_core/proto.dart' as pb;
+import 'package:flutter/material.dart';
+
+class MainProgression extends StatefulWidget {
+  const MainProgression({
+    Key? key,
+    required this.releaseState,
```

Comment:
Requiring releaseState and making it nullable are counter to each other, choose one

Reply:
I am a little confused. There is a possibility that `releaseState` is null. But we do require it to be passed to the widget no matter what. If we only make `releaseState` nullable and not required, then there could be a possibility that a non-null `releaseState` is not passed to the widget.

Reply:
Done